sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/surveillance/SurveillanceDashboardSteps.java

```diff
@@ -0,0 +1,75 @@
+package org.sormas.e2etests.steps.web.application.dashboard.surveillance;
+
+import cucumber.api.java8.En;
+import javax.inject.Inject;
+import org.assertj.core.api.SoftAssertions;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pages.application.dashboard.Surveillance.SurveillanceDashboardPage;
+
+public class SurveillanceDashboardSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  private final SoftAssertions softly;
+  private Integer covid19DiseaseCounterIntValueBefore;
+  private Integer covid19DiseaseCounterIntValueAfter;
+  private Integer newCasesCounterIntValueBefore;
+  private Integer newCasesCounterIntValueAfter;
+
+  @Inject
+  public SurveillanceDashboardSteps(WebDriverHelpers webDriverHelpers, SoftAssertions softly) {
+    this.webDriverHelpers = webDriverHelpers;
+    this.softly = softly;
+
+    When(
+        "^I save value for COVID disease counter$",
+        () -> {
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(10);
+          String covid19DiseaseCounterRawValue =
+              webDriverHelpers.getTextFromWebElement(
+                  SurveillanceDashboardPage.COVID19_DISEASE_COUNTER);
+          covid19DiseaseCounterIntValueBefore = Integer.valueOf(covid19DiseaseCounterRawValue);
+        });
+
+    Then(
+        "^I check that previous saved counters have been increment$",
```

Comment:
I would add more precise info about which data we're checking because we have counters in all pages, and this might get duplicated for other tests

Reply:
Yes, good point I will change it to be more precise